Index: if_else/perimeterOfsquare.py
===================================================================
diff --git a/if_else/perimeterOfsquare.py b/if_else/perimeterOfsquare.py
deleted file mode 100644
--- a/if_else/perimeterOfsquare.py	(revision dcec9592a1900b5109aa7eed46ae1d9737a64700)
+++ /dev/null	(revision dcec9592a1900b5109aa7eed46ae1d9737a64700)
@@ -1,3 +0,0 @@
-a=int(input("enter a value"))
-square=4*a
-print("perimeter of square is",square)
\ No newline at end of file
